src/main/java/frc/robot/RobotContainer.java

```diff
@@ -244,6 +263,16 @@ private void configureButtonBindings() {
                 new MagicMotionPos(mainArm, 0, 0, 0),
                 Commands.waitSeconds(.5),
                 new MagicMotionAbsoluteZero(mainArm, shaftEncoder)));
+
+    controllertwo
```

Comment:
Do we need some documentation about bindings?

Reply:
We have some jpgs in the repository about which buttons go to which controls.